### DISCUSSION TITLE
Feature request: Add WriteFreely Logo

### DISCUSSION THREAD

**Discussion by escuco** [JulianPrieber/littlelink-custom#433] [Predefined Links]
Hello!

It would be nice, if the [WriteFreely](https://github.com/writefreely/writefreely) logo could be added to the predefined site list.

![wf-icon](https://github.com/LinkStackOrg/LinkStack/assets/1647405/66cbf294-5b04-455a-a315-42e0cd64271b)

Thank you!

**Comment by escuco** [comment]
Hello!

Is there any chance, that the logo will be in one of the next upgrades?

Cheers
Jan

**Comment by escuco** [comment]
In the latest update v4.5.2 you included some new buttons. Any chance to get this writefreely button with one of the next update or is something against it?

**Comment by JulianPrieber** [comment]
We currently only accept submissions by users.
You can open a pull requests for the buttons you want added and we'll add them in the next version.

**Reply by escuco** [reply]
Ok, thanks for clarification.
